src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -146,14 +150,13 @@ class PaymentInformation extends React.Component {
 
   render() {
     const {
+      directDepositIsSetUp,
       isEligible,
       isLoading,
       multifactorEnabled,
       paymentInformation,
+      paymentAccount,
     } = this.props;
-    const directDepositIsSetUp =
```

Comment:
This value is now passed in directly using a selector so the logic for where to look in the Redux state is encapsulated in the selector. A bonus is that we're getting rid of this `get()` call.